src/scenes/LatestList/LatestListView.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+class LatestListView extends Component {
+
+    constructor(props) {
```

Comment:
rewrite on functional component
you don't need state for map products

Reply:
done